### PR TITLE
feat(curriculum): adding content for HTML forms and tables review page

### PR DESCRIPTION
**Pull request by jdwilkin4**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Screenshot 2024-10-29 at 8 49 36 AM

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/review-html-tables-and-forms/671a872c17382a582e455c4c.md [hunk 1]
```diff
@@ -9,7 +9,43 @@ dashedName: review-html-tables-and-forms
...
 
+## Working with HTML form elements and attributes
 
+- **`Form` element**: used to create an HTML form for user input.
+- **`action` attribute**: used to specify the URL where the form data should be sent.
+- **`Input` element**: used to create an input field for user input.
+- **`type` attribute**: used to specify the type of input field. Ex. text, password, email, etc.
+- **`Label` element**: used to create a label for an input field.
+- **`for` attribute**: used to specify which input field the label is for.
+- **implicit form association**: inputs can be associated with labels by wrapping the input field inside the `label` element.
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/review-html-tables-and-forms/671a872c17382a582e455c4c.md:20]:
```suggestion
- **Implicit form association**: inputs can be associated with labels by wrapping the input field inside the `label` element.
```

File: curriculum/challenges/english/25-front-end-development/review-html-tables-and-forms/671a872c17382a582e455c4c.md [hunk 2]
```diff
@@ -9,7 +9,43 @@ dashedName: review-html-tables-and-forms
...
+- **explicit form association**: inputs can be associated with labels by using the `for` attribute on the `label` element.
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/review-html-tables-and-forms/671a872c17382a582e455c4c.md:21]:
```suggestion
- **Explicit form association**: inputs can be associated with labels by using the `for` attribute on the `label` element.
```

**Review by naomi-lgbt — COMMENTED** [review]
Aside from code values that would be lower-case, I think list items should start with a capital letter.

**Review by naomi-lgbt — APPROVED** [review]

**Comment by jdwilkin4** [comment]
TODO: merge latest changes from main and make necessary updates to this PR to rename cert to full stack

**Comment by moT01** [comment]
A PR has been merged that [moves the block order from the `meta.json` file to the `full-stack.json` file.](https://github.com/freeCodeCamp/freeCodeCamp/pull/56795) This PR will need an update, and the order will need to be removed from the `meta.json` file and moved to the new file.

**Review by moT01 — APPROVED** [review]

**Pull request merged**